rust/CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 # Changelog
 
+# 0.11.5
+
+* Updates from mediasoup TypeScript `3.11.9..=3.12.16`.
+
 # 0.11.4
 
 * Fix consuming data producer from direct transport by data consumer on non-direct transport.
 
 # 0.11.3
 
-* Updates from mediasoup TypeScript `3.10.13..=3.11.8`.
+* Updates from mediasoup TypeScript `3.11.1..=3.11.8`.
```

Comment:
`3.11.3` was alrady covered in the previous release, this should have been `3.11.3` I think

Reply:
Typo. Fixed.